.github/workflows/action-infrastructure-stack.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       - name: "Derive TF Version"
         id: derive_tf_version
-        uses: NHSDigital/uec-dos-management/.github/actions/derive-tf-version@v0.9
+        uses: NHSDigital/uec-dos-management/.github/actions/derive-tf-version@v0.11
```

Comment:
Surely the v11 changes are already in main ? I'd expect only changes to the one workflow to turn on the cleardown